packages/web-react/src/components/TooltipModern/stories/TooltipModern.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { SpiritTooltipModernProps } from '../../../types';
+import ReadMe from '../../Tooltip/README.md';
+import { Button } from '../..';
+import { TooltipModern, TooltipTrigger, TooltipPopover } from '../../Tooltip';
+
+const Placements = [
+  'top',
+  'bottom',
+  'left',
+  'right',
+  'left-end',
+  'left-start',
+  'right-end',
+  'right-start',
+  'top-end',
+  'top-start',
+  'bottom-end',
+  'bottom-start',
+];
+
+const meta: Meta<typeof TooltipModern> = {
+  title: 'Components/TooltipModern',
+  component: TooltipModern,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+    layout: 'centered',
+  },
+  argTypes: {
+    children: {
+      control: 'object',
+    },
+    id: {
+      control: 'text',
+    },
+    placement: {
+      control: 'select',
+      options: Object.values(Placements),
+      table: {
+        defaultValue: { summary: 'bottom' },
+      },
+    },
+  },
+  args: {
+    children: <>TooltipPopover</>,
+    id: 'TooltipModernExample',
+    enableFlipping: true,
+    enableShifting: true,
+    enableSizing: true,
+    enableFlippingCrossAxis: true,
+    placement: 'bottom',
+    flipFallbackPlacements: ['bottom', 'left', 'right', 'top'],
```

Comment:
Actually, the new features never apply in Storybook because even the smallest viewport is too large…

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/eefe14b6-6339-4236-a6f9-d51c56c55108)

Possible improvements:

- make tooltip text longer, like in other demos,
- enlarge the "canvas" so it forces overflow at least on the smallest Storybook viewports.

Reply:
storybook demo updated